api/platform/platform_controller.go

```diff
@@ -136,6 +143,7 @@ func (c *Controller) patchPlatform(request *web.Request) (*web.Response, error)
 		return nil, errDecode
 	}
 	newPlatform.ID = platformID
+	newPlatform.CreatedAt = time.Time{}
```

Comment:
Won't this override the original creation time?

Reply:
No. This will set `CreatedAt` to the zero value of `time.Time` which means it will be skipped in the update query. The idea here is to prevent users from modifying the `CreatedAt` field.

Reply:
Got it 👍

Reply:
this is not very reliable though - at some point we would want to be able to update certain fields to their default values (e.x. clean up description)